internal/pkg/ociimage/transport.go

```diff
@@ -120,20 +120,6 @@ func TransportOptionsFromSystemContext(sc *types.SystemContext) *TransportOption
 	return &tOpts
 }
 
-// SystemContextFromTransportOptions returns a containers/image SystemContext
-// initialized from a TransportOptions struct. If the TrasnportOptions is nil,
-// then nil is returned.
-//
-// Deprecated: for containers/image compatibility only. To be removed in
-// SingularityCE v5
-func SystemContextFromTransportOptions(tOpts *TransportOptions) *types.SystemContext {
```

Comment:
Note - this has not been in a stable release... it was marked as a Deprecated method when added in an intermediate step of the ggcr re-factoring... and is no longer needed.